docs/ReleaseHistory.md

```diff
@@ -11,6 +11,16 @@
 - FPS => False positive reduction in static analysis.
 - FNS => False negative reduction in static analysis.
 
+# UNRELEASED
+- DEP: `System.Buffers` dependency has been removed. The package no longer has any dependencies.
+- BRK: .NET 6.0 and 7.0 are no longer supported as they have reached end-of-life. Use a [supported version version of .NET](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core): .NET 8.0 or greater. 
+- BRK:  .NET Framework 4.5.1 through 4.6.0 are no longer supported. Use a version of .NET Framework version that supports [.NET Standard 2.0](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0): NET 4.6.1 or greater with .NET 4.7.2 or greater strongly recommended. Note that there are no supported versions of Windows that have a version of .NET Framework that would be affected by this change at runtime, but build changes may be required.
+- BRK: `IdentifiableScan` no longer supports stream input.
```

Comment:
>IdentifiableScan [](http://example.com/codeflow?start=8&length=16)

Is there any API surface area change that came with this? I can answer my own question in a moment, of course, when I look at that file. :)

Reply:
OK, I answered it. :) So things like `IdentifiableScan(byte[], long) have gone bye-bye. It's a good practice to exhaustively describe all the public API we remove explicitly in our release notes.